src/makers/win32/appx.js

```diff
@@ -65,7 +65,10 @@ export default async (dir, appName, targetArch, forgeConfig, packageJSON) => { /
     opts.devCert = await createDefaultCertificate(opts.publisher, outPath);
   }
 
-  if (!opts.publisher.match(/^CN=/)) {
+  if (typeof opts.publisher.match !== 'function') {
```

Comment:
Okedoke; if most people think that is more readable then it is probably more readable :) and I'm probably pedantically ridiculous for thinking about the possibility that some fool might do `delete String.prototype.match` so I will update this PR with your suggestion.